pkg/spec/spec.go

```diff
@@ -173,6 +174,12 @@ type DeploymentSpecMod struct {
 	ext_v1beta1.DeploymentSpec `json:",inline"`
 }
 
+type JobSpecMod struct {
+	ControllerFields      `json:",inline"`
+	batch_v1.JobSpec      `json:",inline"`
```

Comment:
Shouldn't this have some comment tags for JsonSchema generations as described in [spec.go conventions](https://github.com/kedgeproject/kedge/blob/master/docs/development.md#specgo-conventions
)?
ping @surajssd

Reply:
done, added